scripts/seedDb.mjs

```diff
@@ -51,3 +56,25 @@ async function recreateLeagueData(leagueKeyPrefix, dataRepo) {
         await redis.json.set(leagueKeyPrefix+user.userId, "$", userString)
     }
 }
+
+// Create league configuration data
+
+await recreateLeagueConfigurationData("league_configuration:amazing_race:35:", amazingRace35LeagueConfiguration)
+await recreateLeagueConfigurationData("league_configuration:amazing_race:36:", amazingRace36LeagueConfiguration)
+await recreateLeagueConfigurationData("league_configuration:big_brother:26:", bigBrother26LeagueConfiguration)
+await recreateLeagueConfigurationData("league_configuration:survivor:47:", survivor47LeagueConfiguration)
```

Comment:
nit: I would maybe move these calls up near where we are adding user data.

Reply:
Updated :)